Makefile

```diff
@@ -13,6 +13,10 @@ ifeq ($(filter $(MAKECMDGOALS),config clean),)
 	endif
 endif
 
+AZURE_ENV_FILE := $(shell azd env list --output json | jq -r '.[] | select(.IsDefault == true) | .DotEnvPath')
```

Comment:
Maybe this should be above the `ENV_FILE := .env` line and if it errors, it defaults to `.env` so if someone hasn't configured azd yet that everything fails gracefully. You could then feed the filename into the existing logic that includes and exports - just a thought

Reply:
The ui tests will not run successfully unless the azd env is configured anyway. The failure is graceful already - it just ignores the including of the azure env file and goes ahead to run the tests. If the env variables are available from any source, the cypress command will pick them up.